src/cql2ccl.c

```diff
@@ -83,7 +83,7 @@ static void pr_term(const char **cpp, int stop_at_space,
     *cpp = cp;
 }
 
-static int node(struct cql_node *cn,
```

Comment:
Why have you renamed this function as well as `bool`?

Reply:
Because if `bool` is renamed to `node_bool` (I think this name is not-bad choice), I think the original `node` function should be renamed similarly.  
(New function have `node_` prefix, so leaving original `node` function name is too confusing in my opinion.)

If you have better idea how to rename `bool` function, I can let you do so.

Reply:
Fair enough.